UnityProject/Assets/Scripts/Core/Database/PersistenceServer.cs

```diff
@@ -59,7 +63,20 @@ public static async Task<ApiResult<AccountGetCharacterSheets>> GetAccountsCharac
 
 		public static async Task<ApiResult<JsonObject>> DeleteAccountsCharacterByID(int id, string token)
 		{
-			var response = await ApiServer.Delete<JsonObject>(GetUri($"characters/{id}/delete"), token);
+			var response = await ApiServer.Delete<JsonObject>(GetUri($"characters/{id}/delete",""), token);
+			try
```

Comment:
this try block is interesting. When would the logging throw?

Reply:
left over code, removed